examples/cisco/demo_upgrade.pp

```diff
@@ -69,7 +68,7 @@
     }
 
     cisco_upgrade { 'image' :
-      version           => "${version}",
+      package           => "${image}",
```

Comment:
As we discussed combine `package` and `source_uri` into a single property that can manage the device.

The output of the `puppet resource` command show display `package` information and `version` information.